Userland/Utilities/nc.cpp

```diff
@@ -42,18 +45,48 @@ int main(int argc, char** argv)
     bool should_listen = false;
     bool verbose = false;
     bool should_close = false;
+    bool udp_mode = false;
     const char* addr = nullptr;
     int port = 0;
 
     Core::ArgsParser args_parser;
     args_parser.set_general_help("Network cat: Connect to network sockets as if it were a file.");
     args_parser.add_option(should_listen, "Listen instead of connecting", "listen", 'l');
     args_parser.add_option(verbose, "Log everything that's happening", "verbose", 'v');
+    args_parser.add_option(udp_mode, "UDP mode", "udp", 'u');
     args_parser.add_option(should_close, "Close connection after reading stdin to the end", nullptr, 'N');
     args_parser.add_positional_argument(addr, "Address to connect to or listen on", "address");
     args_parser.add_positional_argument(port, "Port to connect to or listen on", "port");
     args_parser.parse(argc, argv);
 
+    if (udp_mode) {
+        if (should_listen) {
+            fprintf(stderr, "listening on UDP not yet supported\n");
+            exit(1);
```

Comment:
```suggestion
            return 1;
```

(just for consistency, as the same is done below)